### DISCUSSION TITLE
Pearson's residuals for Scanorama

### DISCUSSION THREAD

**Discussion by Jay-Leung** [brianhie/scanorama#136] [Q&A]
Hello! I am new to Scanpy and Scanorama, so I am not sure if this is the correct place to ask. 

I am trying to normalize my scRNA-Seq data (3 datasets) with Pearson's residuals (https://genomebiology-biomedcentral-com.libproxy1.nus.edu.sg/articles/10.1186/s13059-021-02451-7) in the experimental module of Scanpy, followed by integration with Scanorama. However, given that each dataset should have batch specific normalization factors, I should preprocess them and normalize separately before using Scanorama right? 

Would it be possible to feed in Pearson's residuals into Scanorama?

Thanks!

**Comment by brianhie** [comment]
Hi @Jay-Leung, apologies for the delayed response. I think initially, you should try to see if having very light normalization would be appropriate for you analysis, as there are competing views on the usefulness of different normalization schemes. But in theory, you can apply Scanorama to normalized data.
